src/app/coursegrab/__init__.py

```diff
@@ -1,12 +1,22 @@
 from flask import Blueprint
-from app.coursegrab.controllers.hello_world_controller import *
 from app.coursegrab.controllers.create_user_controller import *
+from app.coursegrab.controllers.hello_world_controller import *
+from app.coursegrab.controllers.login_controller import *
+from app.coursegrab.controllers.oauth2_callback_controller import *
 from app.coursegrab.controllers.track_course_controller import *
+from app.coursegrab.controllers.update_session_controller import *
 
 # CourseGrab Blueprint
 coursegrab = Blueprint("coursegrab", __name__, url_prefix="/api")
 
-controllers = [HelloWorldController(), CreateUserController(), TrackCourseController()]
+controllers = [
```

Comment:
What does controller mean? Is it a standard to naming specific type of classes?

Reply:
standard for naming files which house the endpoint logic